tests/conftest.py

```diff
@@ -404,6 +404,21 @@ def test_omim_term(request):
     return omim_term
 
 
+@pytest.fixture
```

Comment:
You already have test orpha diseases in another fixture, [here](https://github.com/Clinical-Genomics/scout/blob/93b6bba54708e15d5319f230db1d6dc94d90ff40/tests/parse/conftest.py#L45). Perhaps move the latter fixture outside the parsing tests so that you can invoke it from your test (test_query_omim)?

Reply:
I have moved the test diseases, but I'm not sure it got less messy since the datatypes and formatting of the information needed for adding a term to the database and for testing the parsing functions are not the same. Now I have opted to move everything to this file and then updated the fixtures to return the expected format for one specific disease.